lambda/src/firebase-authorizer/firebase-authorizer.js

```diff
@@ -116,29 +117,26 @@ const checkUserAuthorization = async event => {
   }
 }
 
+const _handler = async event => {
+  initNFA({
+    firebaseProjectId: process.env.LAMBDA_FIREBASE_PROJECT_ID,
+    firebasePrivateKey: decryptedFirebasePrivateKey.replace(/\\n/g, '\n'),
+    firebaseClientEmail: process.env.LAMBDA_FIREBASE_CLIENT_EMAIL,
+    firebaseDatabaseURL: process.env.LAMBDA_FIREBASE_DATABASE_URL,
+    firebasePublicAPIKey: process.env.FIREBASE_PUBLIC_API_KEY,
+    cookieKeys: [await decryptValue(process.env.COOKIE_SECRET_20220711)],
+  })
+  return checkUserAuthorization(event)
+}
+
 const handler = async event => {
   // Decrypt secure environment variables.
   if (decryptedFirebasePrivateKey) {
```

Comment:
Suggestion (ignore as desired): you might add a single helper that verifies NFA has been initialized and that all necessary decrypted values are set, and if not, deals with doing those things.